tests/test-helper.js

```diff
@@ -9,4 +10,12 @@ setApplication(Application.create(config.APP));
 
 setup(QUnit.assert);
 
+setRunOptions({
+  rules: {
+    'scrollable-region-focusable': {
```

Comment:
I don't seem to have a good control/understanding over this rule. In development and production, I can use the arrow keys to scroll up and down the page, but the page seems to violate this rule in testing.